Dockerfile

```diff
@@ -23,6 +23,7 @@ FROM ubuntu:focal
 
 MAINTAINER "tomkralidis@gmail.com"
 
+ARG WIS2NODE_PIP3_EXTRA_PACKAGES
```

Comment:
What if `WIS2NODE_PIP3_EXTRA_PACKAGES` is not defined (no plugins)?  The below will just run `pip3 install` (which will result in error)?

Perhaps something like:
```
diff --git a/Dockerfile b/Dockerfile
index d4ad16c..f29abfe 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -29,6 +29,13 @@ ENV TZ="Etc/UTC" \
 
 COPY . /app
 
+ENV WIS2NODE_PIP3_EXTRA_PACKAGES=None
+
+RUN if [ "$WIS2NODE_PIP3_EXTRA_PACKAGES" = "None" ]; \
+    then export WIS2NODE_PIP3_EXTRA_PACKAGES=echo; \
+    else export WIS2NODE_PIP3_EXTRA_PACKAGES=pip3 install ${WIS2NODE_PIP3_EXTRA_PACKAGES}; \
+    fi
+
 # FIXME: install newer version of eccodes
 # FIXME: csv2bufr: remove and install from requirements.txt once we have a stable release
 # FIXME: pygeometa: remove and install from requirements.txt once we have a stable release
@@ -44,6 +51,8 @@ RUN apt-get update -y \
     # install wis2node
     && cd /app \
     && python3 setup.py install \
+    # install wis2node plugins, if defined
+    && $PIP_PLUGIN_PACKAGES \
     # cleanup
     && apt-get remove --purge -y ${BUILD_PACKAGES} \
     && apt autoremove -y  \

```

Reply:
Yes but we need to mount WIS2NODE_PIP3_EXTRA_PACKAGES as a build arg not an ENV. It is declared as an argument in docker-compose.yml which would probably be a better place to put the default option.